### DISCUSSION TITLE
Is systemd-boot unavailable when installing it in Virtualbox

### DISCUSSION THREAD

**Discussion by c02y** [archlinux/archinstall#2906] [Q&A] [answered]
I cannot find systemd-boot in the boot options, there are only Grub and Limine for me, is it because I try to install it in VirtualBox?

**Comment by Torxed** [comment, marked as answer]
It's because you haven't enabled EFI mode on your VM. systemd-bootd only support EFI

**Reply by c02y** [reply]
After I enabled EFI, systemd-boot is the default option, thanks for the explanation.
